rails6/en/chapter07-placing-orders.adoc

```diff
@@ -261,7 +261,7 @@ class Api::V1::OrdersController < ApplicationController
   before_action :check_login, only: %i[index]
 
   def index
-    render json: OrderSerializer.new(current_user.orders).serializable_hash
+    render json: OrderSerializer.new(current_user.orders).serializable_hash.to_json
```

Comment:
Is this needed? I think Rails convert `hash` to JSON automatically ?

Reply:
Oh, this is needed when using the JsonAPI serializer gem.

Here’s a look at the documentation from the gem docs that explains it: https://github.com/jsonapi-serializer/jsonapi-serializer#migrating-from-netflixfast_jsonapi